kaggle-classification/tf_trainer/run.py

```diff
@@ -76,4 +67,4 @@ def main():
 
 
 if __name__ == "__main__":
-  main()
+  tf.app.run(main)
```

Comment:
How does this differ from just calling main()?

Reply:
got to do it this way to use tf.app.flags (same as Google's absl library)